robot/impl/utils.go

```diff
@@ -0,0 +1,33 @@
+package robotimpl
+
+import (
+	"context"
+	"testing"
+
+	"go.viam.com/test"
+
+	"go.viam.com/rdk/config"
+	"go.viam.com/rdk/logging"
+	"go.viam.com/rdk/robot"
+)
+
+// TODO: this duplicates `robotimpltest.LocalRobot` for tests that are in the `robotimpl`
```

Comment:
whats the TODO here?

Reply:
removing the duplicate helper, if possible - here's some context on how I tried (and failed) to do that: https://github.com/viamrobotics/rdk/pull/3795#discussion_r1560086601

Reply:
I will add a ticket we can link this TODO to